pkg/model/components/etcdmanager/model.go

```diff
@@ -256,6 +256,17 @@ func (b *EtcdManagerBuilder) buildPod(etcdCluster kops.EtcdClusterSpec, instance
 	}
 
 	{
+		// Remap all init container images via AssetBuilder
+		if len(pod.Spec.InitContainers) > 0 {
```

Comment:
There is no need to check the length of a slice before a loop.
This should be moved after the `etcdSupportedVersions` loop and do all init containers in a single loop.

Reply:
Thanks for the suggestions. I've made those changes.